lib/Raven/Middleware/ExceptionInterfaceMiddleware.php

```diff
@@ -36,6 +41,7 @@
     public function __construct(Client $client)
     {
         $this->client = $client;
+        $this->configuration = $client->getConfig();
```

Comment:
It seems to me that the client instance is not needed anymore, so we could pass directly the configuration as argument of the constructor and this way we would even remove a circular reference

Reply:
It's used once. I mean to do a small refactor later to eliminate it, but I'm not sure how\where to move the used function (`translateSeverity`)